src/token.spec.js

```diff
@@ -33,7 +38,7 @@ describe('token', () => {
     it('creates a token string which can be used to sign in', async () => {
       const token = await createToken()
 
-      await getConnection({credentials: {token}})
+      xo1 = await getConnection({credentials: {token}})
```

Comment:
Yes,you're right it's better to close it here same thinks with the second connection. I will fix it.